scripts/install-tests.sh

```diff
@@ -53,6 +53,40 @@ if [ "${TESTING_PROFILE}" = "singularity-tests" ]; then
     singularity --version
 fi
 
+if [ "${TESTING_PROFILE}" = "singularity-tests" ]; then
```

Comment:
Why did this file change? I don't see any commits in the hotfix branch that modified it.

Reply:
Fixed; it looks like I somehow duplicate the block